packages/swingset-liveslots/src/liveslots.js

```diff
@@ -1357,16 +1351,8 @@ function build(
     vatGlobals,
   });
 
-  function setVatOption(option, value) {
+  function setVatOption(option, _value) {
```

Comment:
I find it kind of weird that `setVatOption` is now essentially a no-op, since as of this PR we no longer have any per-vat options that can be set.  Part of me says we should just remove this entirely, but I'm concerned that we'd have to reinvent it if at some future time we discover a vat option that we need to add.  On the other hand, if that never happens we keep carrying this strange vestigial bit with us.  I'm thinking maybe a comment is in order.

Reply:
Yeah, I'm also hesitant to rip it out. I'll add a comment.